### PR TITLE
chore: Update to to capacitor v4

### PR DESCRIPTION
**Pull request by MateiNenciu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,11 +18,11 @@
   "author": "mesur.io <oss@mesur.io>",
   "license": "MIT",
   "dependencies": {
-    "@capacitor/core": "^3.0.0"
+    "@capacitor/core": "^4.0.1"
```
Comment (rdlabo) [inline package.json:21]: For thinking the future, I think Capacitor versions should not be fixed except for major versions.

**Comment by rdlabo** [comment]
Pre-released with next tag:

https://github.com/capacitor-community/firebase-analytics/releases/tag/v2.0.0-1

This is a pre-release for those who want to use this plugin with Capacitor4 in a hurry.

@maintainer Released with next tag to make the world compatible with Capacitor v4. Please check out this pull request and decide if you want to merge or not. Thanks for all your great work, and we look forward to working with you in the future.

**Comment by vanessag** [comment]
Any updates on when this will be merged in and new version released for capacitor 4? Thanks!

**Comment by mukul-myhq** [comment]
Hi @rdlabo , thanks for the plugin, having the same issue. Is there any update on this?

**Comment by Hambat** [comment]
How about cap 4?

**Comment by 24jr** [comment]
not to be repetitive but I too would like this to work for cap 4 as soon as possible

**Comment by robingenz** [comment]
Since this repository currently has no maintainer, I'd like to say that everyone is welcome to try the [Capacitor Firebase Analytics](https://github.com/capawesome-team/capacitor-firebase/tree/main/packages/analytics) plugin from Capawesome in the meantime, which supports Capacitor 4.
Thanks to all who have worked on this plugin so far!

**Pull request merged**